boards/common/samd21-arduino-bootloader/Makefile.include

```diff
@@ -6,6 +6,11 @@ ifeq ($(PROGRAMMER),bossa)
   # by default, we use BOSSA to flash this board and take into account the
   # preinstalled Arduino bootloader. ROM_OFFSET skips the space taken by
   # such bootloader.
-  ROM_OFFSET ?= 0x2000
+  ifeq ($(CPU),samd5x)
+    ROM_OFFSET ?= 0x4000
+  else
+    ROM_OFFSET ?= 0x2000
+  endif
   BOSSA_ARDUINO_PREFLASH = yes
+  PREFLASH_DELAY ?= 2
```

Comment:
Why do you need to set this value here ?

Reply:
This makes flashing actually reliable on my machine.
The default 1s is way to short, I experienced the same issue on the samd21 `serpente` board.

Reply:
Maybe add this change in a separate commit ?

Reply:
there you go 1d638bf0176b6a7db50110161d3cf249b56452a4